### PR TITLE
Add descriptions about cloud storage sink

### PR DESCRIPTION
**Pull request by zhaoxinyu**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -645,6 +686,25 @@ dispatchers = [
 # 对于 MQ 类的 Sink，可以指定消息的协议格式
 # 目前支持 canal-json、open-protocol、canal、avro 和 maxwell 五种协议。
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:687]:
```suggestion
# 对于下游类型是 Kafka、S3、ABS、GCS、NFS 的情况，可以指定消息的协议格式
# 目前支持 canal-json、avro 和 csv 三种协议。
```

**Review by leoppro — APPROVED** [review]

**Review by leoppro — COMMENTED** [review]
LGTM

File: ticdc/ticdc-csv.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+"U","employee","hr",433305438660591630,102,"Alex","Alice","2018-06-15","Beijing"
+```
+
+## 数据类型映射
+
+| MySQL 类型                                           | CSV 类型 | 示例                          | 描述                                   |
+|-----------------------------------------------------|----------|------------------------------|---------------------------------------|
+| BOOLEAN/TINYINT/SMALLINT/INT/MEDIUMINT/BIGINT       | Integer  | 123                          |                                       |
+| FLOAT/DOUBLE                                        | Float    | 153.123                      |                                       |
+| NULL                                                | Null     | \N                           |                                       |
```
Comment (shichun-0415) [inline ticdc/ticdc-csv.md:86]:
```suggestion
| BOOLEAN/TINYINT/SMALLINT/INT/MEDIUMINT/BIGINT       | Integer  | 123                          |  -                                     |
| FLOAT/DOUBLE                                        | Float    | 153.123                      |  -                                     |
| NULL                                                | Null     | \N                           | -                                      |
```

File: ticdc/ticdc-csv.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+| TIMESTAMP/DATETIME                                  | String   | "1973-12-30 15:30:00.123456" | 格式: yyyy-MM-dd HH:mm:ss.%06d         |
+| DATE                                                | String   | "2000-01-01"                 | 格式: yyyy-MM-dd                       |
+| TIME                                                | String   | "23:59:59"                   | 格式: HH:mm:ss                         |
+| YEAR                                                | Integer  | 1970                         |                                       |
```
Comment (shichun-0415) [inline ticdc/ticdc-csv.md:90]:
```suggestion
| YEAR                                                | Integer  | 1970                         |  -                                     |
```

File: ticdc/ticdc-csv.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+| VARCHAR/JSON/TINYTEXT/MEDIUMTEXT/LONGTEXT/TEXT/CHAR | String   | "test"                       | 以 UTF-8 编码输出                       |
+| VARBINARY/TINYBLOB/MEDIUMBLOB/LONGBLOB/BLOB/BINARY  | String   | "6Zi/5pav"                   | 以 base64 编码输出                      |
+| BIT                                                 | Integer  | 81                           |                                       |
+| DECIMAL                                             | String   | "129012.1230000"             |                                       |
+| ENUM                                                | String   | "a"                          |                                       |
+| SET                                                 | String   | "a,b"                        |                                       |
```
Comment (shichun-0415) [inline ticdc/ticdc-csv.md:96]:
```suggestion
| BIT                                                 | Integer  | 81                           | -                                      |
| DECIMAL                                             | String   | "129012.1230000"             | -                                      |
| ENUM                                                | String   | "a"                          | -                                     |
| SET                                                 | String   | "a,b"                        | -                                     |
```

File: ticdc/manage-ticdc.md [hunk 5]
```diff
@@ -792,6 +846,207 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+    - ColumnLength: 该列的长度。详见 [数据类型](/ticdc/manage-ticdc.md#数据类型)。
+    - ColumnPrecision: 该列的精度。详见 [数据类型](/ticdc/manage-ticdc.md#数据类型)。
+    - ColumnScale: 该列小数位的长度。详见 [数据类型](/ticdc/manage-ticdc.md#数据类型)。
+    - ColumnNullable: 值为 true 时表示该列可以含 NULL 值。
+    - ColumnIsPk: 值为 true 时表示该列是主键的一部分。
+- TableColumnsTotal: TableColumns 数组的大小。
+
+### 数据类型
+
+数据类型定义为 T(M[, D])，详见 [数据类型概述](/data-type-overview.md#数据类型概述)。
```
Comment (qiancai) [inline ticdc/manage-ticdc.md:954]:
```suggestion
数据类型定义为 `T(M[, D])`，详见[数据类型概述](/data-type-overview.md#数据类型概述)。
```

File: ticdc/ticdc-csv.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
+---
+title: TiCDC CSV Protocol
+summary: 了解 TiCDC CSV Protocol 的概念和使用方法。
+---
+
+# TiCDC CSV Protocol
+
+## 使用 CSV
+
+当使用云存储作为下游 sink 时，你可以使用 CSV 格式将 DML 事件发送到下游云存储。
```
Comment (qiancai) [inline ticdc/ticdc-csv.md:10]:
```suggestion

当使用云存储作为下游 sink 时，你可以使用 CSV 格式将 DML 事件发送到下游云存储。

## 使用 CSV

```

Comment (qiancai) [reply ticdc/ticdc-csv.md:10]: reason: 文档标题下一般需要有一句话整体介绍下这篇文档

File: ticdc/manage-ticdc.md [hunk 7]
```diff
@@ -792,6 +846,207 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+
+元数据信息以 JSON 格式存储到如下的文件中：
+
+```shell
+{
+    "checkpoint-ts":433305438660591626
+}
+```
+
+- checkpoint-ts: commit-ts 小于等于此 checkpoint-ts 的事务都被写入下游存储当中。
```
[the remark this answers is not in the document]
Comment (zhaoxinyu) [reply ticdc/manage-ticdc.md:889]: 是的

File: ticdc/manage-ticdc.md [hunk 8]
```diff
@@ -645,6 +686,25 @@ dispatchers = [
...
+# 是否使用 partition 作为分隔字符串。
+enable-partition-separator = false
+
+# 从 v6.4 开始，TiCDC 支持以 CSV 格式将数据变更记录保存至云存储中。
+[sink.csv]
+# 字段之间的分隔符。必须为 ASCII 字符，默认值为 `,`
+delimiter = ','
+# 用于围住字段的引号字符。空值代表不使用引号字符。默认值为 `"`。
+quote = '"'
+# csv 列为 null 时将被表示的字符。默认值为 `\N`
```
[the remark this answers is not in the document]
Comment (zhaoxinyu) [reply ticdc/manage-ticdc.md:698]: 意思是列为 null 时 csv 列将显示什么字符

[the remark this answers is not in the document]
Comment (zhaoxinyu) [reply ticdc/manage-ticdc.md:698]: 改成 "csv 列为 null 时以什么字符来表示" 如何？

File: ticdc/manage-ticdc.md [hunk 9]
```diff
@@ -645,6 +686,25 @@ dispatchers = [
...
+# 文件路径的日期分隔类型。可选类型有 `none`、`year`、`month`、`day`。默认值为 `none`，即不使用日期分隔。
+date-separator = 'none'
+# 是否使用 partition 作为分隔字符串。
+enable-partition-separator = false
+
+# 从 v6.4 开始，TiCDC 支持以 CSV 格式将数据变更记录保存至云存储中。
+[sink.csv]
+# 字段之间的分隔符。必须为 ASCII 字符，默认值为 `,`
+delimiter = ','
+# 用于围住字段的引号字符。空值代表不使用引号字符。默认值为 `"`。
```
[the remark this answers is not in the document]
Comment (zhaoxinyu) [reply ticdc/manage-ticdc.md:696]: 可以改成 "包裹"

File: ticdc/ticdc-csv.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+
+## 事务性约束
+
+- 单个 CSV 文件中后一行数据的 commit-ts 必须大于前一行数据的 commit-ts。
+- 单表的同一事务不能跨不同的 CSV 文件来存储。
+- 相同事务涉及的不同表可跨不同的 CSV 文件来存储。
+
+## 数据格式定义
+
+CSV 行中的每一列定义如下：
```
[the remark this answers is not in the document]
Comment (zhaoxinyu) [reply ticdc/ticdc-csv.md:43]: 这个地方就是指单行中的每一列数据

File: ticdc/manage-ticdc.md [hunk 11]
```diff
@@ -848,6 +832,8 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 
 ## 云存储类 Sink
 
+本部分详细介绍云存储类 Sink。
```
Comment (shichun-0415) [inline ticdc/manage-ticdc.md:836]: 二级标题`云存储类 Sink` 和三级标题 `数据变更记录`、`元数据`、`DDL 事件` 以及`数据类型` 是什么关系呢？

File: ticdc/ticdc-csv.md [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
```
Comment (shichun-0415) [inline ticdc/ticdc-csv.md:41]:
```suggestion
CSV 文件中，单行的每一列定义如下：
```

File: ticdc/manage-ticdc.md [hunk 13]
```diff
@@ -314,6 +314,25 @@ dispatchers = [
 
 集成具体步骤详见 [TiDB 集成 Confluent Platform 快速上手指南](/ticdc/integrate-confluent-using-ticdc.md)。
 
+#### Sink URI 配置 `s3`
+
+TiCDC 从 v6.4 开始支持将行变更事件保存至 S3 中。
```
Comment (shichun-0415) [inline ticdc/manage-ticdc.md:319]:
```suggestion
TiCDC 从 v6.5.0 开始支持将行变更事件保存至 S3 中。
```

File: ticdc/manage-ticdc.md [hunk 14]
```diff
@@ -642,9 +661,29 @@ dispatchers = [
...
+
+# 以下三个配置项仅在云存储类的 Sink 中使用。
+# 换行符。默认值为空，表示使用 "\r\n" 作为换行符。
+terminator = ''
+# 文件路径的日期分隔类型。可选类型有 `none`、`year`、`month`、`day`。默认值为 `none`，即不使用日期分隔。
+date-separator = 'none'
+# 是否使用 partition 作为分隔字符串。
+enable-partition-separator = false
+
+# 从 v6.4 开始，TiCDC 支持以 CSV 格式将数据变更记录保存至云存储中。
```
Comment (shichun-0415) [inline ticdc/manage-ticdc.md:677]:
```suggestion
# 从 v6.5.0 开始，TiCDC 支持以 CSV 格式将数据变更记录保存至云存储中。
```

File: ticdc/manage-ticdc.md [hunk 15]
```diff
@@ -792,6 +831,209 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+- `table`：`table` 为表名。例如：`s3://bucket/bbb/ccc/test/table1`
+- `table-version-separator`：将文件路径按照表的版本进行分隔。例如：`s3://bucket/bbb/ccc/test/table1/9999`
+- `partition-separator`：将文件路径按照表的分区号进行分隔。例如：`s3://bucket/bbb/ccc/test/table1/9999/20`
+- `date-separator`：将文件路径按照事务提交的日期进行分隔。date-separator 可选值如下：
+    - `none`：不以 date-separator 分隔文件路径。例如：`test.table1` 版本号为 9999 的所有文件都存到 `s3://bucket/bbb/ccc/test/table1/9999` 路径下。
+    - `year`：以事务提交的年份分隔文件路径。例如：`s3://bucket/bbb/ccc/test/table1/9999/2022`。
+    - `month`：以事务提交的年份和月份分隔文件路径。例如：`s3://bucket/bbb/ccc/test/table1/9999/2022-01`。
+    - `day`：以事务提交的年月日来分隔文件路径。例如：`s3://bucket/bbb/ccc/test/table1/9999/2022-01-02`。
+- `num`：存储数据变更记录的目录下文件的序号。例如：`s3://bucket/bbb/ccc/test/table1/9999/2022-01-02/CDC000005.csv`。
+- `extension`：文件的扩展名，v6.4 只支持 csv 格式。
```
Comment (shichun-0415) [inline ticdc/manage-ticdc.md:858]:
```suggestion
- `extension`：文件的扩展名，v6.5.0 只支持 csv 格式。
```

**Pull request closed without merging**